content/python/concepts/casting/casting.md

```diff
@@ -113,95 +112,14 @@ Traceback (most recent call last):
 ValueError: invalid literal for int() with base 10: 'seven'
 ```
 
-### Type Casting `float` to `str`
-
-Below is illustration of casting of float data type to string where `float()` function take `x` as argument.
-
-```py
-x = 7.2
-
-y = str(x) # Casting into string
-
-print(type(x))
-print(y)
-print(type(y))
-```
-
-This will output:
-
-```shell
-<class 'float'>
-7.2
-<class 'str'>
-```
-
-### Type Casting `str` to `float`
-
-Let's see the example of casting of string data type to float. In below example `x` is being of string data type is casted to float where `float()` function take `x` as argument.
-
-```py
-x = "7.2"
+## Operations on Different Types of Data
 
-# Casting into float
-y = float(x)
-
-print(type(x))
-print(y)
-print(type(y))
-```
-
-This will output:
-
-```shell
-<class 'str'>
-7.2
-<class 'float'>
-```
-
-### Type Casting `int` to `float`
-
-Below is an example to show how an int variable is converted to float. In below example `float()` function take `x` as argument where `x` in integer type data.
-
-```py
-x = 7
-
-# Casting into float
-y = float(x)
-
-print(type(x))
-print(y)
-print(type(y))
-```
-
-This will output:
-
-```shell
-<class 'int'>
-7.0
-<class 'float'>
-```
-
-### Type Casting `float` to `int`
-
-Similarly like above example , in this float data type variable is casted to integer where `int()` function take x as argument.
+When operating on data, it is important to keep their type in mind. The following Codebyte is a flawed attempt to print the square of a number specified by the user. Try running it and ponder the result.
```

Comment:
```suggestion
When operating on data, it is important to keep their type in mind. The following codebyte is a flawed attempt to print the square of a number specified by the user. Try running it and ponder the result.
```

Reply:
I don't believe we want to place flawed codebytes in Docs. @sonnynomnom can this be confirmed?

Rather, we can condense all the deleted code snippets and add a more complex example in there as well.